app/src/main/java/com/android/sample/model/profile/MockProfilesRepository.kt

```diff
@@ -0,0 +1,89 @@
+package com.android.sample.model.profile
```

Comment:
good mocking of the profileRepository that will be used for tests and good error handling. (addLikedActivity and removeLikedActivity need to be implemented later).